### DISCUSSION TITLE
Where is `RenderNodes` type in next.js plugin

### DISCUSSION THREAD

**Discussion by schultzp2020** [markdoc/markdoc#116] [Q&A] [answered]
Currently I am trying to access the `MarkdocNextJsPageProps.markdoc.content` type; however, when I look at the type declaration it refers to `RenderNodes`. 

``` js
// next.js/src/index.d.ts
import type {Options, RenderNodes, Schema} from '@markdoc/markdoc';

export type MarkdocNextJsPageProps = {
  markdoc?: {
    content: RenderNodes;
    frontmatter: Record<string, any>;
    file: {
      path: string;
    };
  };
};
```

I have tried to locate the type declaration for `RenderNodes` in '@markdoc/markdoc', but I cannot find it. I have also tried to directly access it by `type RenderNodes = MarkdocNextJsPageProps['markdoc']['content']`, but that does not add type support. Where can I find the type delcaration for `RenderNodes`?

**Comment by mfix-stripe** [comment, marked as answer]
Apologies @schultzp2020, the Next.js plugin is slightly out of date. The correct type is `RenderableTreeNodes`, which you can import from `@markdoc/markdoc`.

I just released a new version of `@markdoc/next.js` ([`0.1.6`](https://github.com/markdoc/next.js/releases/tag/0.1.6)) that includes this fix.
